TYPO3.Flow/Classes/TYPO3/Flow/Cache/Backend/SimpleFileBackend.php

```diff
@@ -396,4 +395,31 @@ public function rewind()
             $this->cacheFilesIterator->next();
         }
     }
+
+    /**
+     * @param string $cacheEntryPathAndFilename
```

Comment:
you and your return voids.... ;)

Can we remove that from the coding guideline? Or put it as CAN HAVE instead of MUST